en/GLOSSARY.md

```diff
@@ -23,6 +23,7 @@ See sections [Compare things](./python_introduction/#compare-things) and [Boolea
 The front-end framework we use to [make our page pretty](./css).
 
 # class
+In object-oriented programming, a class is an extensible program-code-template for creating objects. It defines a set of properties & methods that are common to all objects of one type.
```

Comment:
This seems more based on the general "class" concept in programming (the first sentence is probably paraphrased from [Wikipedia article "Class (computer programming)"](https://en.wikipedia.org/wiki/Class_%28computer_programming%29), isn't it?) than on [Python's specific concept of classes](https://docs.python.org/3/tutorial/classes.html).

(That's alright with me, but we should be aware of that.)